src/github/logic.py

```diff
@@ -288,6 +288,9 @@ def maybe_automerge_pull_request_and_rerun_stale_checks(
             pull_request.is_mergeable() and pull_request.is_approved()
         )
 
+    logger.info(
+        f"Pull request status: test {pull_request.is_build_successful()}, mergeable {pull_request.is_mergeable()}, approved {pull_request.is_approved()}"
```

Comment:
Can you prepend this with the id too? Also `test` is probably difficult to understand out of context considering the value it likely to be `true` or `false`. These logs should also be well-formed to serve other companies (whether they use SGTM or not) so it would be better to have more explicit logs